Kitodo-DataManagement/src/main/resources/db/migration/V2_48__Create_table_typeGenerate_task_x_folder.sql

```diff
@@ -0,0 +1,29 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+--
+-- Migration: Create table generateContents_task_x_folder
+
+-- 1. Create table generateContents_task_x_folder
+
+CREATE TABLE generateContents_task_x_folder (
```

Comment:
Please read coding conventions for Table names: https://intranet.slub-dresden.de/display/KITPROD/SQL-Schema-Namenskonventionen#SQL-Schema-Namenskonventionen-Table
They should start with a capital letter.

Reply:
Maybe this documentation is outdated or was never folowed, but the database is only using lowercase letters and camelCase starting with a lowercase letter, see also: https://github.com/kitodo/kitodo-production/blob/master/Kitodo-DataManagement/src/main/resources/db/migration/V2_0__Translate_tables_names_English.sql

I agree with only using lowercase/ lower camelCase, because tables are case sensitive in Linux but case-insensitive in Windows which will lead to confusing errors.